### ISSUE TITLE
Clicking "Dismiss Difference" [X] button does nothing + spelling error in tooltip

### ISSUE THREAD

**Issue by sampellino** [moshfeu/vscode-compare-folders#145]
1.) When I click the "X" button next to the filename in the "Differences" menu, the file does not get removed from the list of files.
2.) "Dissmiss" should be "Dismiss" (one 's')
![image](https://github.com/moshfeu/vscode-compare-folders/assets/5458241/5f0b6b70-4fa9-459b-9677-0eb1c74368f3)

**Comment by moshfeu** [comment]
Thanks for reporting 🙏
I'll check what's going on with the button and of course awill fix the typo

**Comment by sampellino** [comment]
> Thanks for reporting 🙏 I'll check what's going on with the button and of course awill fix the typo

Of course! Fantastic extension, I use it nearly daily at my job. Let me know if you need any help, and I can clone the repo to try to fix the issue I'm seeing.

**Comment by moshfeu** [comment]
Wow, that's amazing! Thanks for sharing
I'd be happy if you can fix these. It's alway more fun with collaboration
Let me know how it goes and if there is anything I can assist

**Comment by moshfeu** [comment]
By the way, I verified that it's working on my mac. What's your environment?

**Comment by sampellino** [comment]
Windows 11, latest OS / VSCode versions.

**Comment by sampellino** [comment]
I'm installing the dependencies on my personal machine to modify the extension and test it. So far I just have the spelling mistakes corrected, but I'll try to get the local clone of the extension running in VSCode to verify it's fixed. My hunch is that it's something in `filterIgnoredFromDiffs()` that is causing the issue.

**Comment by sampellino** [comment]
Looks like within `dismissDifference = async (e: TreeItem) => {`,  `path` is possibly missing the drive letter at the beginning of the string

![image](https://github.com/moshfeu/vscode-compare-folders/assets/5458241/83544c6e-32eb-40c7-966a-5bd64749797f)

**Comment by moshfeu** [comment]
fixed by #146 

Thanks @sampellino for both the issue and the solution, you're the best  💪
